web-components/src/components/menu-overlay/MenuOverlay.ts

```diff
@@ -413,6 +414,10 @@ export namespace MenuOverlay {
       }
     }
 
+    private focusTheNestedTrigger(triggerEleement?: HTMLElement) {
```

Comment:
Function name can be focusOnTheNestedTrigger and correct the spelling of triggerElement .

Reply:
done